### DISCUSSION TITLE
https://vuepress-theme-hope.gitee.io/v2/zh/config/plugins/intro.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#526] [Announcements]
# https://vuepress-theme-hope.gitee.io/v2/zh/config/plugins/intro.html

vuepress-theme-hope 会调用很多 VuePress 插件。 一些插件会自动启用，如果你不需要，可以在主题选项中将其禁用; 一些插件只有你在进行必要配置后才会启用。 注 作为 VuePress Org 的成员之一，Mr.Hope 开发了很多 VuePress 插件。 vuepress-theme-hope 调用的所有插件均为官方插件或托管在 vuepress-theme-hope 仓库下由 Mr.Hope 自己开发的插件。 官方插件的文档详见 [VuePress2 官网][vuepress] Mr.Hope 自己开发的插件也全部拥有自己的文档，可以搭配其他主题使用。

https://vuepress-theme-hope.gitee.io/v2/zh/config/plugins/intro.html

**Comment by XiaoYuer2022** [comment]
棒👍🏻！
